doc/6/getting-started/.vuejs/cypress/support/commands.js

```diff
@@ -0,0 +1,99 @@
+function reinitialisation() {
+  const kuzzle = Cypress.env('kuzzle');
+  
+  // Clear collection
+  return cy.request({
+    url: `http://${kuzzle.host}:${kuzzle.port}/${kuzzle.index}/${kuzzle.collection}/_truncate`,
+    method: 'DELETE',
+  })
+    .then(searchResponse => {
+      cy.log(`Request : truncate ${kuzzle.collection} status : ${searchResponse.status}`);
+      cy.wait(500);
+    });
+}
+
+Cypress.Commands.add('createMessage', (body) => {
+  const kuzzle = Cypress.env('kuzzle');
+  return cy.request({
+    url: `http://${kuzzle.host}:${kuzzle.port}/${kuzzle.index}/${kuzzle.collection}/_create`,
+    method: 'POST',
+    body: body,
+  })
+    .its('body')
+    .then(response => {
+      cy.log(`Create status : ${response.status} {${body.username} / ${body.value}}`);
+      cy.wait(500);
+    });
+});
+
+
+Cypress.Commands.add('initialisation', () => {
+  const kuzzle = Cypress.env('kuzzle');
+
+  // Delete index if exists
+  cy.request({
```

Comment:
Promises must either be returned or have their own `catch` handler.

(this remark must be applied to all created promises)